sci-libs/libxc/libxc-4.3.4.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools fortran-2
+
+DESCRIPTION="A library of exchange-correlation functionals for use in DFT"
+HOMEPAGE="http://octopus-code.org/wiki/Libxc"
+SRC_URI="https://gitlab.com/libxc/libxc/-/archive/${PV}/${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="fortran static-libs -test"
+
+pkg_setup() {
+	use fortran && fortran-2_pkg_setup
+}
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf --enable-shared \
+		$(use_enable fortran) \
+		$(use_enable static-libs static)
```

Comment:
disable static libs

Reply:
It seems to work afaict. Why disable it?